api/src/main/java/io/kafbat/ui/config/ClustersProperties.java

```diff
@@ -99,6 +108,16 @@ public static class SchemaRegistryAuth {
   public static class TruststoreConfig {
     String truststoreLocation;
     String truststorePassword;
+    boolean verifySsl = true;
```

Comment:
- [ ] add this property to contracts
- [ ] update docs to include this property